docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -54,7 +52,7 @@ the risk of network disruption. You must enforce the changes by setting the
 172.31.0.0/24 via 192.168.122.100 dev br-ctlplane
 ----
 
-. To configure similar routes for the existing subnet for the control plane services that are attached to the networks, add `routes` entries to the `NodeNetworkConfigurationPolicy` CRs for each network. For example:
+. Link local routes to existing deployment should also be configured on {rhos_long} worker nodes. This is achieved by adding `routes` entries to the `NodeNetworkConfigurationPolicy` CRs for each network. For example:
```

Comment:
```suggestion
. You also must configure link local routes to existing deployment on {rhos_long} worker nodes. This is achieved by adding `routes` entries to the `NodeNetworkConfigurationPolicy` CRs for each network. For example:
```
Should it be "RHOSO" worker nodes, or "RHOCP" worker nodes?

Reply:
RHOSO. I believe this is what `rhos_long` stands for.

Reply:
Yes, you're right.